ecosystem/platform/server/app/controllers/application_controller.rb

```diff
@@ -19,10 +19,13 @@ def set_csrf_cookie
       value: form_authenticity_token,
       secure: true,
       same_site: :strict,
-      domain: ENV.fetch('SITE_DOMAIN', 'localhost')
```

Comment:
Can we delete the entire `set_csrf_cookie` method (if not, why just delete this line)?

Reply:
deleted the line as it would fail to set in some cases (i.e domain mismatch with ngrok, etc; just easier to not have it)

Reply:
we don't read it anywhere anymore I see... if we have turbolinks POST, will we need to resurrect it? I'll delete for now